src/components/Tool/Appeal/InitialPage/Appeal.tsx

```diff
@@ -105,7 +105,7 @@ const Appeal = ({
               className={classes.nameLink}
             >
               <NextLink
-                href={`/accountLists/${accountListId}/tools/appeals/appeal/${id}`}
+                href={`/accountLists/${accountListId}/tools/appeals/appeal/${id}/list`}
```

Comment:
You've removed this in a different PR I believe

Reply:
Correct, I temporarily added the `/list` to the end because otherwise it was interpreting the appeal id as the contact id. In #1354 I make `ContactPanelProvider` more robust to not need the `/list` suffix. I initially did it this way to keep this PR smaller since I was going to work on making appeals work with `ContactPanelProvider` in a different PR.